pkg/controller/secretannotator/azure/reconciler.go

```diff
@@ -0,0 +1,112 @@
+package azure
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	controllerName      = "secretannotator"
```

Comment:
Should this use the const you made public in pkg/controller/secretannotator/ ?

Reply:
done